ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/graphql/GraphQLHttpService.java

```diff
@@ -244,8 +249,8 @@ public String url() {
   }
 
   // Facilitate remote health-checks in AWS, inter alia.
-  private void handleEmptyRequest(final RoutingContext routingContext) {
-    routingContext.response().setStatusCode(201).end();
+  private void handleEmptyRequestAndRedirect(final RoutingContext routingContext) {
+    routingContext.reroute(GRAPH_QL_ROUTE);
```

Comment:
I think BESU-58 is asking for a HTTP protocol redirect, not treating "/" as equivalent to "/graphql".   Maybe a 308 instead because of post/get changes - https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/301 https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/308

Reply:
Added a 308 redirect instead.